src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,67 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const linksAmount = Math.ceil(total / perPage);
+  const linkNumbers = getNumbers(1, linksAmount);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          onClick={() => onPageChange(currentPage - 1)}
+        >
+          «
+        </a>
+      </li>
+
+      {linkNumbers.map(linkNumber => {
+        return (
+          <li
+            className={`page-item ${currentPage === linkNumber ? 'active' : ''}`}
+            key={linkNumber}
+          >
+            <a
+              data-cy="pageLink"
+              className="page-link"
+              href={`#${linkNumber}`}
+              onClick={() => onPageChange(linkNumber)}
```

Comment:
Ensure that the `href` attribute for pagination links points to a valid location or is omitted since you are handling the click event with JavaScript. The current `href` value `#{linkNumber}` might not be a valid URL fragment for your application.